docs/analyzer-config.md

```diff
@@ -7,8 +7,9 @@ The following configurations can be set in a `.editorconfig` file to configure p
 | `natls.style.comparisons` | `sign`, `short`, `false` | [`NL006`](../tools/ruletranslator/src/main/resources/rules/NL006)|
 | `natls.style.disallowtoplevelvars` | `true`, `false` | [`NL018`](../tools/ruletranslator/src/main/resources/rules/NL018)|
 | `natls.style.qualifyvars` | `true`, `false` | [`NL019`](../tools/ruletranslator/src/main/resources/rules/NL019)|
-| `natls.style.discourage_independent` | `true`, `false` | [`NL028`](../tools/ruletranslator/src/main/resources/rules/NL028)|
-| `natls.style.discourage_gitmarkers` | `true`, `false` | [`NL030`](../tools/ruletranslator/src/main/resources/rules/NL030)|
+| `natls.style.discourageindependent` | `true`, `false` | [`NL028`](../tools/ruletranslator/src/main/resources/rules/NL028)|
```

Comment:
`discourage_independent` and `discourage_gitmarkers` shouldn't be changed, that would be a breaking change.